package_creator.py

```diff
@@ -130,9 +130,12 @@ def get_code_file(package_path, script_type):
     :rtype: str
     """
 
-    ignore_regex = r'^CommonServerPython\.py|^CommonServerUserPython\.py|demistomock\.py|test_.*\.py|_test\.py'
+    ignore_regex = r'CommonServerPython\.py|CommonServerUserPython\.py|demistomock\.py|test_.*\.py|_test\.py'    
     if not package_path.endswith('/'):
         package_path += '/'
+    if package_path.endswith('Scripts/CommonServerPython/'):
```

Comment:
shouldn't this be `startswith`?

Reply:
It's ok. We get a full path of the sort `/usr/.../Scripts/CommonServerPython`